### ISSUE TITLE
Customizing directory names

### ISSUE THREAD

**Issue by mbrandonw** [facebook/ios-snapshot-test-case#202]
Right now my test images are being stored in `{Framework_Name}.{View_Test_Name}`, and it's super verbose. Is there anyway to omit the framework name?

**Comment by Grubas7** [comment]
Did you check README, Installation with CocoaPods, point 2?

**Comment by mbrandonw** [comment]
Yeah, as far as I can tell that allows me to select the root directory that all the test sub-directories are held, but not actually rename the subdirectories. So in my case I have a bunch of directories in `ReferenceImages/_64/{Framework_Name}.{View_Test_Name}`, where I would prefer to have `ReferenceImages/_64/{View_Test_Name}`.

Apologies if I have misunderstood, but still not quite sure how to handle this.
